cg/constants/housekeeper_tags.py

```diff
@@ -91,6 +91,10 @@ class HkMipAnalysisTag:
     SAMPLE_INFO: list[str] = ["sample-info"]
 
 
+class HkNFAnalysisTags:
```

Comment:
To follow the convention for other workflows
```suggestion
class NFAnalysisTags:
```